### PR TITLE
[O2B-666]Only admins should be allowed to create tags

### PR DESCRIPTION
**Pull request by melyamalfoy**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.38.0](https://github.com/AliceO2Group/Bookkeeping/releases/tag/%40aliceo2%2Fbookkeeping%400.39.0)
+* Notable changes for user:
+  * Only administrators can create tags 
+  * Only admniistrators can edit tags
```
Comment (martinboulais) [inline CHANGELOG.md:8]: typo, `admiinistrators`

File: .dockerignore [hunk 2]
```diff
@@ -7,4 +7,4 @@
 !spec/
 !.sequelizerc
 !.nycrc
-!package*.json
+!package*.json
```
Comment (martinboulais) [inline .dockerignore:10]: I think you should revert this change, it is not useful

File: lib/public/views/Tags/Create/index.js [hunk 3]
```diff
@@ -33,17 +33,17 @@ const createTagScreen = (model) => {
...
-            h('input#text.form-control.w-25.mb2', {
-                placeholder: 'Enter the tag content...',
-                minlength: 2,
-                maxlength: 20,
-                value: text,
-                oninput: (e) => {
-                    model.tags.text = e.target.value;
-                },
-            }, text),
             model.isAdmin() && [
```
Comment (martinboulais) [inline lib/public/views/Tags/Create/index.js:36]: As you mentioned, you should add an error message stating that the user do not have rights to create a tag, because here for a non admin it looks a bit like a bug

File: lib/public/views/Tags/Details/index.js [hunk 4]
```diff
@@ -83,7 +83,7 @@ const tagDetails = (model) => {
                                     },
                                 }, 'Cancel'),
                             ]
-                            : [
+                            : model.isAdmin() && [
```
Comment (martinboulais) [inline lib/public/views/Tags/Details/index.js:86]: There is the test l.69 which already check that

File: lib/public/views/Tags/Overview/index.js [hunk 5]
```diff
@@ -31,9 +31,12 @@ const tagOverview = (model) => h('', { onremove: () => model.tags.overview.reset
 const showTagsTable = (model, tags) => [
     h('.flex-row.justify-between.items-center', [
         h('h2', 'Tags'),
-        h('button.btn.btn-primary.w-15#create', {
-            onclick: () => model.router.go('/?page=tag-create'),
-        }, 'Create tag'),
+
```
Comment (martinboulais) [inline lib/public/views/Tags/Overview/index.js:34]: This newline is not necessary

**Review by martinboulais — CHANGES_REQUESTED** [review]

**Review by melyamalfoy — COMMENTED** [review]
improved based on feedback

**Review by martinboulais — DISMISSED** [review]

**Review by martinboulais — APPROVED** [review]
